qemu/tests/image_creation_luks_with_non_utf8_secret.py

```diff
@@ -0,0 +1,40 @@
+import os
+import re
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import utils_misc
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    Luks image creation with non_utf8_secret:
+    1. It should be failed to create the image.
+    2. The error information should be corret.
+       e.g. Data from secret sec0 is not valid UTF-8
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    image_stg = params["images"]
+    image_stg_name = params["image_name_stg"]
+    root_dir = data_dir.get_data_dir()
+    image_stg_path = utils_misc.get_path(root_dir, image_stg_name)
+    if os.path.exists(image_stg_path):
+        os.remove(image_stg_path)
+    err_info = params["err_info"]
+    non_utf8_secret = params["non_utf8_secret"]
+    params["image_secret_stg"] = process.system_output(non_utf8_secret,
+                                                       shell=True)
+    stg = QemuImg(params.object_params(image_stg), root_dir, image_stg)
```

Comment:
what about stg.remove() here, so line 24,26-28 can be removed

Reply:
Agree with you. Updated, thanks.